kano/utils.py

```diff
@@ -31,6 +31,9 @@ def run_cmd(cmd, localised=False):
     if not localised:
         env['LC_ALL'] = 'C'
 
+    if unsudo and 'SUDO_USER' in os.environ and os.environ['SUDO_USER'] != 'root':
```

Comment:
We actually already have a function in this file ([`get_user_unsudoed()`](https://github.com/KanoComputing/kano-toolset/blob/d12e1b60c6c0e852f41932561f3043bf311a961d/kano/utils.py#L173-L179)) for retrieving the unsudo'd user - maybe we should use that instead?